src/yoroi-wallets/cardano/shelley/__snapshots__/transactionCache.test.ts.snap

```diff
@@ -13,7 +13,12 @@ Object {
   "0a8962dde362eef1f840defe6f916fdf9701ad53c7cb5dd4a74ab85df8e9bffc": Object {
     "blockHash": "fb418acaa29c66e799a16b594f7beedfe2ef53413e9863b61a418f2df1ff1442",
     "blockNum": 2909238,
-    "certificates": Array [],
+    "certificates": Array [
```

Comment:
why did this change?

Reply:
To test the changes in the variable `perRewardAddressCertificates`

Reply:
Also, `transactionCache.test.ts` and `transactionManager.test.ts` are using the same mocks now.